### PR TITLE
Report sessionization metrics to OpenTSDB

### PR DESCRIPTION
**Pull request by scabbage**
@optimizely/data-services 

This is to add sessionization specific metrics to OpenTSDB. Samza periodically publishes metrics to a Kafka topic (will be a separate PR on `samza-jobs`). This tcollector consumes from that metrics topic and sends formatted messages to OpenTSDB.

### CODE REVIEW

File: collectors/0/samza_prod.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python
+import sys
+
+from collectors.lib.samza_metric_reporter import SamzaMetricReporter
+
+CONSUMER_GROUP_ID = "tcollector_samza_prod"
+
+KAFKA_BOOTSTRAP_SERVERS = [
```
Comment (mikecdavis) [inline collectors/0/samza_prod.py]: It's a bit weird to leak this type of stuff into a publicly forked repo. Can we use ENV variables or something?

**Comment by mikecdavis** [comment]
Ideally this type of configuration would be done in chef vs this repo. Can we move to ENV variables?

File: collectors/0/samza_prod.py (hunk 1 above)
Comment (scabbage) [inline collectors/0/samza_prod.py]: Can you elaborate the concern here? This forked repo is private, right?

**Comment by scabbage** [comment]
Since we don't have a separate staging monitoring server, we will have to use two ENV vars - `KAFKA_PROD_BOOTSTRAP_SERVERS` and `KAFKA_STAGING_BOOTSTRAP_SERVERS`.

**Comment by scabbage** [comment]
Added some todos. Merge now.

**Pull request merged**